plugin_test.go

```diff
@@ -1,13 +1,27 @@
 package main
 
 import (
+	"log"
 	"os"
 	"os/exec"
 	"testing"
 
 	. "github.com/franela/goblin"
+	"github.com/joho/godotenv"
 )
 
+func loadEnv(keyValue string) {
```

Comment:
It seems this function takes a string, uses that string to write a env file, then uses that env file to load/populate the env.

Is there a reason we don't just directly set the env?  I don't think we need to test the `godotenv` lib itself which is what I feel like we are sort of doing here.